irods/test/data_obj_test.py

```diff
@@ -1463,6 +1465,49 @@ def test_register_with_checksum(self):
         # delete file
         os.remove(test_file)
 
+
+    def test_object_names_with_nonprintable_chars (self):
+        test_dir = helpers.irods_shared_tmp_dir()
+        loc_server = self.sess.host in ('localhost', socket.gethostname())
+        if not(test_dir) and not(loc_server):
+            self.skipTest('data_obj register requires server has access to local or shared files')
+        temp_names = []
+        vault = ''
+        try:
+
+            #import pdb;pdb.set_trace() # dwm_
+
+            fname = re.sub( r'[/\n]', '',
+                            ''.join(map(chr,range(1,128))) )
+            prefix_ = 'object_'  + fname
+            resc_name = 'regWithNonPrintableNamesResc'
+            vault = make_ufs_resc_in_tmpdir(self.sess, resc_name, allow_local = loc_server)
+            def enter_file_into_irods ( session, filename, method, **options ):
```

Comment:
whitespace before opening parens here, but no whitespace in others.

is that on purpose?

Reply:
corrected